packages/manager/modules/nutanix/src/dashboard/general-info/constants.js

```diff
@@ -3,6 +3,26 @@ export const TRAVAUX_LINK = {
   DEFAULT: 'https://www.status-ovhcloud.com/',
 };
 
+export const GUIDE_PACKAGES_URL = {
+  CA: 'https://www.ovhcloud.com/en-ca/hosted-private-cloud/nutanix/packaged',
+  QC: 'https://www.ovhcloud.com/fr-ca/hosted-private-cloud/nutanix/packaged',
+  DE: 'https://www.ovhcloud.com/de/hosted-private-cloud/nutanix/packaged',
+  ES: 'https://www.ovhcloud.com/es-es/hosted-private-cloud/nutanix/packaged',
+  FR: 'https://www.ovhcloud.com/fr/hosted-private-cloud/nutanix/packaged',
+  GB: 'https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix/packaged',
+  IE: 'https://www.ovhcloud.com/en-ie/hosted-private-cloud/nutanix/packaged',
+  IT: 'https://www.ovhcloud.com/it/hosted-private-cloud/nutanix/packaged',
+  NL: 'https://www.ovhcloud.com/nl/hosted-private-cloud/nutanix/packaged',
+  PL: 'https://www.ovhcloud.com/pl/hosted-private-cloud/nutanix/packaged',
+  PT: 'https://www.ovhcloud.com/pt/hosted-private-cloud/nutanix/packaged',
+  MA: 'https://www.ovhcloud.com/fr-ma/hosted-private-cloud/nutanix/packaged',
+  SN: 'https://www.ovhcloud.com/fr-sn/hosted-private-cloud/nutanix/packaged',
+  TN: 'https://www.ovhcloud.com/fr-tn/hosted-private-cloud/nutanix/packaged',
+  WS: 'https://www.ovhcloud.com/es/hosted-private-cloud/nutanix/packaged',
+  US: 'https://us.ovhcloud.com/hosted-private-cloud/nutanix/packaged',
+  DEFAULT: 'https://www.ovhcloud.com/en/hosted-private-cloud/nutanix/packaged',
```

Comment:
**nitpick:** This is not a valid link. Maybe you can use `https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix/packaged`